Week-1/index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="/style.css" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Lato"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+
+    <title>Your Page Title</title>
+  </head>
+  <body>
+    <div class="main-div">
+      <nav class="nav">
+        <i class="fa fa-bars" aria-hidden="true"></i>
+        <i class="fa fa-home" aria-hidden="true"></i>
+        <i class="fa fa-search" aria-hidden="true"></i>
+        <i class="fa fa-volume-up" aria-hidden="true"></i>
+        <i class="fa fa-user" aria-hidden="true"></i>
+        <i class="fa fa-spotify" aria-hidden="true"></i>
+        <i class="fa fa-cog" aria-hidden="true"></i>
+        <i class="fa fa-soundcloud" aria-hidden="true"></i>
+      </nav>
+      <section>
+        <header class="page-header">
+          <div class="header-section">
+            <img
+              src="https://uploads.codesandbox.io/uploads/user/a8233562-b36b-483d-9de9-5a1ac148fb0c/2Eq--cattyboard.jpg"
+            />
+            <div class="header-inside">
+              <div>
+                <h1>CattyBoard Top 100 Single Charts(11.06.36)</h1>
+                <p>Unknown Artist</p>
+                <p>2016.Charts.100 songs</p>
+              </div>
+              <div class="header-down">
+                <div>
+                  <p>
+                    <i class="fa fa-play" aria-hidden="true"></i>&nbsp Play all

```

Comment:
&nbsp is used with ; like &nbsp for space, try not to use &nbsp; and manage the space with css property like padding, margin, etc

Reply:
Sure mam